src/common/helpers/export/export-html.js

```diff
@@ -73,45 +36,70 @@ const buildHtml = (data, reference) => {
       const questionsHtml = section.questionAnswers
         .map(({ question, answer }) => {
           return `
-            <div class="question">
-              <div><strong>${question}</strong></div>
-              <div>${answer.displayText.replace(/\n/g, '<br />')}</div>
+            <div class="govuk-summary-list__row">
+              <dt class="govuk-summary-list__key govuk-!-width-one-half govuk-!-font-weight-regular">
+                ${question}
+              </dt>
+              <dd class="govuk-summary-list__value">
+                ${answer.displayText.replace(/\n/g, '<br />')}
+              </dd>
             </div>
             `
         })
         .join('')
 
       return `
-      <div class="section">
-        <h2 class="section-title">${section.title}</h2>
-        ${questionsHtml}
-      </div>
-    `
+        <h2 class="govuk-heading-m">${section.title}</h2>
+        <dl class="govuk-summary-list govuk-!-margin-bottom-9">
+          ${questionsHtml}
+        </dl>
+      `
     })
     .join('')
 
+  const cssPath = path.join(
+    srcFolder,
+    '/common/helpers/export/styles/govuk-frontend-5.10.2.min.css'
+  )
+  const cssContent = fs.readFileSync(cssPath, 'utf-8')
```

Comment:
We need to use the async / await version of this API - otherwise we will be blocking our main thread & causing the performance to drop.